install/create_system_config_cds.md

```diff
@@ -0,0 +1,394 @@
+   >   for more information.
+   > - If the system is using a `cabinets.yaml` file, be sure to update the `cabinets-yaml` field with `'cabinets.yaml'` as its value.
+
+   ```bash
+   vim system_config.yaml
+   ```
+
+## 2. Configure the management network
+
+1. (`pit#`) Change into the `cds` directory.
+
+   ```bash
+   cd "${PITDATA}/prep/cds"
+   ```
+
+1. (`pit#`) Set up the `bond0` network interface as follows:
+
+   - Configure `bond0` interface for single VLAN:
+
+     ```bash
+     "${CVT_PATH}"/cds/network_setup.py --mtlip <ipaddr>,<netmask>,<slave0>,<slave1>
+     ```
+
+   - Configure `bond0` interface for HMN configured network (VLAN 1, 2, and 4):
+
+     ```bash
+     "${CVT_PATH}"/cds/network_setup.py --mtlip <ipaddr>,<netmask>,<slave0>,<slave1> --hmnip <ipaddr>,<netmask>,<vlanid> --nmnip <ipaddr>,<netmask>,<vlanid>
+     ```
+
+     The variables and their descriptions are as follows:
+
+     | Variable  | Description  |
+     | ---       |  ---         |
+     | ipaddr    | IP address   |
+     | netmask   | netmask value |
+     | slave0, slave1 | Network Interface |
+     | vlanid    | VLAN ID number |
+
+     > **NOTE**: The IP address value, netmask value, network interface, and VLAN ID are available in the `system_config.yaml` file in the `prep` directory.
+
+## 3. Configure auto-discovery
+
+### 3.1 Configure Dnsmasq
+
+1. (`pit#`) Configure Dnsmasq and PXE Boot.
+
+   - Configure Dnsmasq for single VLAN:
+
+     ```bash
+     "${CVT_PATH}"/cds/discover_enable.py --iprange <dhcp_ip_start>,<dhcp_ip_end>,<subnet>,<leasetime> --interface bond0
+     ```
+
+     Where, `leasetime` is an optional variable.
+
+     Example Script:
+
+     ```bash
+     "${CVT_PATH}"/cds/discover_enable.py --iprange 10.x.x.x,10.x.x.x,255.x.x.x,24h --interface bond0
+     ```
+
+   - Configure Dnsmasq for HMN configured network (VLAN 1, 2, and 4):
+
+     ```bash
+     "${CVT_PATH}"/cds/discover_enable.py --mtlrange <admin_node_mtl_ip>,<start_ip>,<end_ip> 
+     --nmnrange <admin_node_nmn_ip>,<start_ip>,<end_ip> --hmnrange <admin_node_hmn_ip>,<start_ip>,<end_ip>
+     ```
+
+     Example Script:
+
+     ```bash
+     "${CVT_PATH}"/cds/discover_enable.py --mtlrange 10.x.x.x,10.x.x.x,10.x.x.x  --nmnrange 10.x.x.x,10.x.x.x,10.x.x.x  --hmnrange 10.x.x.x,10..x.x.x,10.x.x.x
+     ```
+
+     > **NOTE**: The `iprange` for single VLAN or `mtliprange`, `nmniprange`, and `hmniprange` for HMN configured network are available in the `system_config.yaml` file in the `prep` directory.
+
+### 3.2 Verify and reset the BMCs
+
+The following steps will verify if IPs are assigned to the BMC nodes and reset the nodes:
+
+1. (`pit#`) If you want to dynamically discover the fabric switches, PDUs, and CMCs, ensure they are set to `dhcp` mode.
+
+1. (`pit#`) List the BMC nodes.
+
+   ```bash
+   "${CVT_PATH}"/cds/discover_start.py -u bmc_username -p bmc_password --list
```

Comment:
Is the user expected to replace `bmc_username` and `bmc_password` with the actual values? Or are those the names of environment variables that have been previously exported? For security reasons, we should not have users passing passwords on command lines. If they need to pass in a password, then they should assign it to an environment variable, export that variable, and then the script can read it from that variable.

Reply:
we will make it interactive for password to be taken to address the security issues.

In the upcoming fixes it would be added